### ISSUE TITLE
A11y_AzureDataStudio_Notebooks - Run_Keyboard:"Filter" button in Results Table is not accessible using keyboard.

### ISSUE THREAD

**Issue by VijayaBharathyA** [microsoft/azuredatastudio#22657]
"[Check out Accessibility Insights! ](https://accessibilityinsights.io/)- Identify accessibility bugs before check-in and make bug fixing faster and easier.”
### GitHub Tags:
#A11yMAS; #A11yTCS; #A11ySev2; #DesktopApp; #BM-AzureDataStudio-Win32-Feb2022; #Azure Data Studio; #Win32; #WCAG2.1.1; #Keyboard; #Benchmark; #Closed #Regressed:04-17-23;

**"Please do not close this bug. This bug should only be closed by TCS, C+AI Accessibility or the Divisional Driver."**

### Environment and OS details:
Azure Data Studio Version: 1.42.0 (user setup)
Windows 11 Enterprise Version	22H2
Installed on	‎28-‎12-‎2022
OS build	22621.1413
Experience	Windows Feature Experience Pack 1000.22639.1000.0
### Pre-Requisite:

1. Install Azure data studio From here https://learn.microsoft.com/en-us/sql/azure-data-studio/download-azure-data-studio?view=sql-server-ver16&tabs=redhat-install%2Credhat-uninstall
2. Once installation done , Install MySQL Extension in Azure data studio
3. Tab Till  new connection and press enter 
4. Enter a valid server name username and password and press connect
### Repro Steps:

1. Launch Azure Data Studio.
2. Open an existing notebook that has code cells in it.
3. Click Run Cells on Notebook toolbar.
4. Tab till Filter
5. Verify that Keyboard focus is not shifting to Filter
### Actual Result:
"Filter" button in Results Table is not accessible using keyboard.
### Expected Result:
Filter button in Results Table should be accessible using keyboard.
### Attachment:

![Video0 (6)](https://user-images.githubusercontent.com/100189407/228870875-9e5a418e-5698-41f9-be78-a74b6a7d2137.gif)

**Comment by VijayaBharathyA** [comment]
Primary bug :https://github.com/microsoft/azuredatastudio-mysql/issues/186

**Comment by alanrenmsft** [comment]
@VijayaBharathyA this is by design, instead, we provided a keyboard shortcut key for it, the default one is "F3" and it is configurable. when focus is in the table, press F3 to will open the menu.

![image](https://user-images.githubusercontent.com/13777222/232166299-562df28f-7501-4925-addd-431fdc629de0.png)

**Comment by VijayaBharathyA** [comment]
Since we are able to access filter using F3 hence closing the bug
![Video0](https://user-images.githubusercontent.com/100189407/232376755-82cadce6-4a3c-4dff-853f-41e4bf5800d8.gif)
